packages/inventory/src/components/PostConfirmationError/tests/__snapshots__/index.test.tsx.snap

```diff
@@ -1,6 +1,6 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`post confirmation Error card page renders and works correctly: card title 1`] = `"Processing error: inventory items failed to be added"`;
+exports[`post confirmation Error card page renders and works correctly: card title 1`] = `"Processing error, inventory items failed to be added"`;
```

Comment:
this seems not to be handled yet, escaping the colon so that it is not interpreted as a namespace separator by i18next